src/main/resources/schema/mutation.graphqls

```diff
@@ -1,5 +1,26 @@
 type Mutation {
-  createWish(input: WishInput!): Wish!
-  deleteWish(id: ID!): Boolean!
-  updateWish(id: ID!, input: WishInput!): Wish
+  "Create a new App Wish "
+  addWish(input: WishInput): Wish
```

Comment:
How about we keep some naming convention? Sorry for I didn't mention it in the task description.
We could go with something like this:

Assuming we have a type called 'something'
- createSomething  
- something, allSomething (for reading)
- updateSomething
- deleteSomething

It uses CRUD verbs - create, update, delete. For READ it use the type name - something, or in case of reading multiple values - allSomething.

Reply:
I guess it's easy to remember (everybody heard of CRUD) and will reduce chances that we have inconsistencies in naming like updateSomething, editSomethingElse (update vs edit) or like comments and wishList (both comments and wishList would return multiple objects, but one use plural form and the second uses 'List' suffix.

Reply:
done :smile: